trash.conf

```diff
@@ -18,6 +18,7 @@ github.com/urfave/cli	v1.18.0-84-g3eb41f1
 golang.org/x/net	4cfeeeb
 golang.org/x/sys	c200b10
 github.com/jpillora/backoff a6f6428
+github.com/bitly/go-simplejson da1a892
```

Comment:
When adding new dependencies please use a tag if possible, in this case `v0.5.0` is what you want.